### PR TITLE
fix(iOS): build SampleApp from vs4m

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?
SampleApp.iOS build on Vs4M is failing when compiling  `Uno.UI.Dispatching` project

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   <PropertyGroup>
-    <TargetFrameworks>xamarinmac20;xamarinios10;MonoAndroid12.0;net461;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>xamarinmac20;xamarinios10;MonoAndroid12.0;netstandard2.0;net461</TargetFrameworks>
 		<TargetFrameworksCI>MonoAndroid11.0;MonoAndroid12.0;xamarinios10;net461;netstandard2.0;xamarinmac20</TargetFrameworksCI>
   </PropertyGroup>
 	
-	<Import Project="../targetframework-override.props" />
+	<PropertyGroup Condition="'$(UnoTargetFrameworkOverride)'!=''">
+		<TargetFrameworks>$(UnoTargetFrameworkOverride)</TargetFrameworks>
+	</PropertyGroup>
```
Comment (jeromelaban) [inline src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: This is very curious, the `	<Import Project="../targetframework-override.props" />` is doing exactly this. What's the error in vs4mac without this ?

Comment (ajpinedam) [reply src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: VS4M is unable to load the dependencies

Comment (ajpinedam) [reply src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: @jeromelaban this is the specifc error on the Package Console:

`/usr/local/share/dotnet/sdk/6.0.101/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: android [/Users/apineda/develop/uno/uno/src/SourceGenerators/System.Xaml/Uno.Xaml.net6.csproj]

/usr/local/share/dotnet/sdk/6.0.101/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To install these workloads, run the following command: dotnet workload install android [/Users/apineda/develop/uno/uno/src/SourceGenerators/System.Xaml/Uno.Xaml.net6.csproj]

/Applications/Visual Studio.app/Contents/Resources/lib/monodevelop/bin/MSBuild/Current/bin/NuGet.targets(352,5): error MSB4181: The "MSBuild" task returned false but did not log an error. [/Users/apineda/develop/uno/uno/src/Uno.UI.sln]

MSBuild exited with code 1
`

Comment (jeromelaban) [reply src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: Interestingly, this means that uno-check needs to be executed to install the proper sdks. Can you try doing that ?

Comment (ajpinedam) [reply src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: Sure!

Comment (ajpinedam) [reply src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: @jeromelaban  ran the `uno-check` and this fixed a few issues on my environment but I am still seeing the same error reported above when using the `<Import Project...`

Comment (jeromelaban) [reply src/Uno.UI.Dispatching/Uno.UI.Dispatching.csproj:10]: Got it. Let's merge it as-is then, thanks!

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**